app/src/androidTest/java/ch/hikemate/app/endtoend/EndToEndTest2.kt

```diff
@@ -47,8 +48,6 @@ class EndToEndTest2 {
     val context = ApplicationProvider.getApplicationContext<Context>()
     FirebaseApp.initializeApp(context)
 
-    auth.createUserWithEmailAndPassword(email, password)
```

Comment:
Question: Why was this removed?

Reply:
Good question! I changed the test to create the user in the UI instead of creating it like this.
I changed it because we need to properly create a profile for this to work.